src/dbt_mcp/telemetry/logging.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import logging
+import os
+from pathlib import Path
+
+FILE_LOGGING_ENV_VAR = "FILE_LOGGING"
```

Comment:
Is `FILE_LOGGING` a good name or should we make it something like `DBT_MCP_SERVER_FILE_LOGGING` so that 
- it is clear that it is for the dbt MCP server
- it doesn't interfere with people potentially already having `FILE_LOGGING` set